config/notifiers.go

```diff
@@ -89,6 +89,20 @@ var (
 		Source:      `{{ template "opsgenie.default.source" . }}`,
 		// TODO: Add a details field with all the alerts.
 	}
+
+	// DefaultPushoverConfig defines default values for Pushover configurations.
+	DefaultPushoverConfig = PushoverConfig{
+		NotifierConfig: NotifierConfig{
+			VSendResolved: true,
+		},
+		Title:            `{{ template "pushover.default.title" . }}`,
+		Message:          `{{ template "pushover.default.message" . }}`,
+		URL:              `{{ template "pushover.default.url" . }}`,
+		Priority:         "2",  // emergency
+		PriorityResolved: "0",  // normal
```

Comment:
I’ve looked at the other notifiers and can’t find a use of templating that uses a conditional on whether the alerts are resolved or not.

Could you please point me to an example of what you’re thinking of?